model/shared/src/main/scala/hmda/model/institution/InstitutionType.scala

```diff
@@ -7,13 +7,13 @@ import enumeratum.{ Enum, EnumEntry }
  */
 sealed trait DepositoryType extends EnumEntry
 
-// FIXME: This could also be modeled as `isDepository: Option[Boolean]`.  Which is cleaner?
 object DepositoryType extends Enum[DepositoryType] {
 
   val values = findValues
 
   case object Depository extends DepositoryType
   case object NonDepository extends DepositoryType
+  case object UndeterminedDepositoryType extends DepositoryType
```

Comment:
To expand on @nickgrippin's statement about data: the reason we have undetermined types is that the HMDA filer population in NIC is a very small subset (like 7,000 of 160,000 when looking at head offices). While typing each entity category is possible it would either require us to incorporate unnecessary types (e.g. edge corporation), create a new schema that encompassed all types in the NIC attributes file, or create an 'other' category.

Happy to discuss this one further. I think it's a by-product of populating the institutions model with all the NIC institutions and not just HMDA filers.